### DISCUSSION TITLE
I am trying to upgrade my antd design but it breaks my project

### DISCUSSION THREAD

**Discussion by eriktoor** [ant-design/ant-design#41446] [Q&A]
I have a legacy project that I am trying to upgrade. 

It is in react and still using create-react-app. It works when I do `npm install --legacy-peer-deps`

And our package.json has these deps

    "@ant-design/compatible": "^1.0.8",
    "@ant-design/icons": "^4.3.0",

And we also install in these ways 

import Icon from "@ant-design/icons";
import "antd/dist/antd.css";

in jsx files

Whenever we try to upgrade versions of antd we get errors from the 

`import "antd/dist/antd.css";` line saying that they don't exist. 

How could I fix this?

**Comment by yoyo837** [comment]
antd@5 now is css in js project. https://ant.design/docs/react/migration-v5

**Reply by eriktoor** [reply]
Thanks for the quick response @yoyo837. Does this mean I can delete these import statements?

**Reply by mzulfanw** [reply]
you can import with ` antd/dist/reset.css` @eriktoor
